src/TestFramework/DataFaker.php

```diff
@@ -195,6 +195,18 @@ public function subscriptionReference()
         return $result;
     }
 
+     /**
+     * Return a subscription status
+     *
+     * @return string
+     */
+    public function subscriptionStatus()
+    {
+        $statuses = array("Active", "Suspended", "Cancelled");
+        shuffle($statuses);
```

Comment:
instead of using shuffle and current, you could do

```php
return $statuses[$this->intBetween(0, count($statuses) - 1)];
```

That way, we can be sure it's using the same seed as everything else in the DataFaker. This is helpful because then if you want to debug something, you can just set an explicit seed in the DataFaker constructor and then every time the tests run they'll use the same values.